### ISSUE TITLE
Any pretrained model?

### ISSUE THREAD

**Issue by deep-practice** [CaoWGG/CenterNet-darknet#1]

**Comment by CaoWGG** [comment]
> _No description provided._

python3 torch2darknet.py will generate dla34_ctdet.cfg  and dla34_ctdet.weights  (pretrained dla34)

**Comment by deep-practice** [comment]
Thanks very much.And does this implementation can achieve the original accuracy ?

**Comment by CaoWGG** [comment]
> Thanks very much.And does this implementation can achieve the original accuracy ?

I haven't finished training.  You can train it.
